ui/components/multichain/pages/connections/components/site-not-connected.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Box, Text } from '../../../../component-library';
+import {
+  AlignItems,
+  BlockSize,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextColor,
+} from '../../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+
+export const SiteNotConnected = () => {
+  const t = useI18nContext();
+  return (
+    <Box
+      data-testid="site-not-connected"
+      display={Display.Flex}
+      flexDirection={FlexDirection.Column}
+      justifyContent={JustifyContent.center}
+      alignItems={AlignItems.center}
+      height={BlockSize.Full}
+      gap={2}
+    >
+      <Text>{t('metamaskNotConnected1')}</Text>
+      <Text color={TextColor.textAlternative}>
+        {t('metamaskNotConnected2')}{' '}
+        <Text color={TextColor.textAlternative} as="strong">
+          {t('metamaskNotConnected3')}
+        </Text>
+      </Text>
+    </Box>
+  );
+};
```

Comment:
Is there a storybook file for this component?

Reply:
Added in [2169209](https://github.com/MetaMask/metamask-extension/pull/21158/commits/2169209d9235af2696a0277eede557a62f8fc56a)